mongodbatlas/resource_mongodbatlas_event_trigger_test.go

```diff
@@ -82,8 +83,8 @@ func TestAccConfigRSEventTriggerDatabase_eventProccesor(t *testing.T) {
 	var (
 		resourceName = "mongodbatlas_event_trigger.test"
 		projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-		awsAccountID = os.Getenv("AWS_ACCOUNT_ID")
-		awsRegion    = os.Getenv("AWS_REGION")
+		awsAccountID = os.Getenv("AWS_EVENTBRIDGE_ACCOUNT_ID")
```

Comment:
i suppose this is a new AWS account id.  I don't see any key, we don't need to authenticate to AWS in these tests? are we really using a real AWS account?

Reply:
Can you elaborate on where you don't see any key?

> we don't need to authenticate to AWS in these tests
as far as I understand, no - and also tested locally and they succeed

> are we really using a real AWS account? 
we have to because of cloud-dev. You can read more at https://jira.mongodb.org/browse/INTMDB-1010

Reply:
ok, sorry i see awsAccID is not the account where the provider is logged in but the event bridge account, that confused me, maybe I keep the variable names from awsAccountID, awsRegion to eventBridgeAccountID, eventBridgeRegion